octodns/provider/ns1.py

```diff
@@ -238,29 +238,80 @@ class Ns1Provider(BaseProvider):
 
     ZONE_NOT_FOUND_MESSAGE = 'server error: zone not found'
 
-    _DYNAMIC_FILTERS = [{
+    _UP_FILTER = {
         'config': {},
         'filter': 'up'
-    }, {
+    }
+
+    _REGION_FILTER = {
         'config': {},
         'filter': u'geofence_regional'
-    }, {
+    }
+    _COUNTRY_FILTER = {
+        'config': {},
+        'filter': u'geofence_country'
+    }
+
+    _SELECT_FIRST_REGION_FILTER = {
```

Comment:
In the UI this is called "Select First Group" might be worth a comment about that. It confused me for a second looking over the changes here.

Reply:
This is a little confusing.
The ns1 api filter name is select_first_region. From NS1 docs:
```
Select First Region
The select_first_region filter simply returns all answers in the same region as the first answer. There is no metadata input or filter config. In the portal this is called “select first group”.
```
I'll add the above as a comment.